src/appmixer/github/list/NewTeam/NewTeam.js

```
@@ -0,0 +1,23 @@
+'use strict';
+const Promise = require('bluebird');
```

Biome (1.9.4)

[error] Do not shadow the global "Promise" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)